llvm/include/llvm/Transforms/Vectorize/SandboxVectorizer/Region.h

```diff
@@ -107,6 +109,25 @@ class Region {
 #endif
 };
 
+/// A pass that runs on a Region.
```

Comment:
Declaring the `RegionPass` class in `Region.h` is a bit strange. Wouldn't it make more sense to move `Pass.*` to the vectorizer too?

Reply:
All the Region-related stuff is currently already in SandboxIR, so "move it into SandboxIR" is just dropping this PR (which is, of course, an option). Or were you talking about something different?